test/utils/format.js

```diff
@@ -148,3 +149,10 @@ test('collectionHrefFromNameVersion returns a nullValue collection if the collec
 test('collectionHrefFromNameVersion returns a nullValue collection if the collection is undefined', function (t) {
   t.is('--', collectionHrefFromNameVersion());
 });
+
+test('fullDate returns the properly formatted date for midnight', function (t) {
+  const date = '2025-04-13T01:15:32.567Z';
+  const formattedDate = fullDate(date);
+
+  t.is(formattedDate, '21:15:32 04/12/25');
+});
```

Comment:
Can you test both '2025-04-13T04:15:32.567Z' (for verifying the ticket) 
and '2025-04-13T01:15:32.567Z'?

Reply:
If you remove Z from the strings and change them to '2025-04-13T00:15:32.567' etc, the unit test will work on all time zones. It currently works locally but fails on ci since the docker probably has utc time.